ci/envoy_build_sha.sh

```diff
@@ -1,3 +1 @@
-# If you edit the SHA here you must also edit the SHA in .circleci/config.yml.
-
-ENVOY_BUILD_SHA=114e24c6fd05fc026492e9d2ca5608694e5ea59d
+ENVOY_BUILD_SHA=$(grep lyft/envoy-build .circleci/config.yml | sed -e 's#.*lyft/envoy-build:\(.*\)#\1#' | head -n 1)
```

Comment:
Is there any way to see if someone put different values inside config.yml? If that's too annoying nbd but thought I would ask.

Reply:
You just want to validate the consistency of all the hashes that appear in that file?

Reply:
Yeah they should all be the same.